content/posts/chapelcon25-retro/index.md

```diff
@@ -0,0 +1,103 @@
+They closed it out with a demo of how to set up and use Chapel on the system. Check out the recording [here](https://youtu.be/5eZCzeObUBc).
+
+[Our second invited talk](https://youtu.be/YuY4XmKxruE) was from LLNL and HPSF's Todd Gamblin on Spack, the open-source HPC package management software. The talk, detailing Spack's path to version 1.0 was full of advice for open-source projects looking to expand their reach. 
+With the perfect mix of technical and social focus, Gamblin's talk was an audience favorite. 
+
+This year, our keynote address came from JuliaLab's Chris Rackauckas and told the story of the rise of scientific machine learning (SciML) in the Julia programming language. 
+The main idea of SciML is combine our scientific models with data sources, enabling efficiency and accuracy at levels unavailable to either source alone.
+This is backed by the technical story of Julia's relentless focus on composability, combined with the non-technical focus on sustainable, open-source development.
+Lessons for the Chapel community and inspiration for libraries and modules abound.
+Check it out [here](https://youtu.be/5ornbvwZJp8).
+
+### More Contributors Than Ever
+
+This year's ChapelCon had a record-breaking 20 presentations from community members across two days.
+The talks covered a wide range of topics, from machine learning and AI, to novel graph algorithms, to performance comparisons, and language interoperability. 
+While there are too many for me to cover here (I encourage you to check them all out [on YouTube](https://www.youtube.com/playlist?list=PLuqM5RJ2KYFgTmbn3PmOocfu5Sh2wEcAb)), I'd like to bring attention to some that were especially enjoyed by the community.
+
+One contribution that succinctly demonstrated Chapel's strengths in delivering both performance and productivity was [Mohammad Dindoost's talk on HiPerMotif](https://youtu.be/acvyLakS6gA).
+HiPerMotif is a hybrid parallel algorithm for identifying subgraph "motifs" within large-scale property graphs. 
+Implemented using Arache (an Arkouda/Chapel-based graph library), it provides up to 66x speedup compared to state-of-the-art methods, and processes graphs large enough to cause memory failures in other technologies. 
+
+The second highlight came from Daniel Fedorin and demonstrates the expansive power of Chapel's type system. 
+Using Chapel's compile-time `param` values and a bit of clever thinking, his approach makes it possible to encode complex data structures and specialized functions, all at compile time. 
+This has wide-reaching implications, including eliminating runtime overhead and expanding compile-time error checking for functions like `printf`. 
+Check out the recording [here](https://youtu.be/-x1zYQSuaT4).
+
+Third, from our very own PC chair Luca Ferranti, was a talk on [automatic differentiation in Chapel](https://youtu.be/ioqxdmSprBM).
```

Comment:
I might drop "very own" as it made me think "from HPE" whereas one of the things I really appreciate about Luca and his work is that he's not.